plugin/index.tpl.html

```diff
@@ -43,7 +44,7 @@ <h1>
 				<a href="https://github.com/[repo]/issues" data-icon="octicon-issue-opened" data-size="large" data-show-count="true" data-count-aria-label="# issues on GitHub" aria-label="Issue [repo] on GitHub">Issues</a>
 				<a href="[github]" data-icon="octicon-star" data-size="large" data-count-href="/[repo]/stargazers" data-show-count="true" data-count-aria-label="# stargazers on GitHub" aria-label="Star [repo] on GitHub">Star</a>
 			</div>
-			<div mv-if="!repo">
+			<div mv-if="!repo and !url">
```

Comment:
Why `and !url`? These buttons only operate on the repo. Also, not sure how we'd have a URL but not a repo, what's the point then?

Unless that's intended for plugins hosted on Gitlab etc?